File: BE/auth-service/src/main/java/com/hogumiwarts/lumos/auth/dto/SignupResponse.java
package com.hogumiwarts.lumos.auth.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
public class SignupResponse {
	private Long memberId;
	private String email;
	private String name;
	private String createdAt;
}

File: Android/data/src/main/java/com/hogumiwarts/data/entity/remote/Response/auth/SignupResponse.kt
package com.hogumiwarts.data.entity.remote.Response.auth

data class SignupResponse(
    val status: Int,
    val message: String,
    val data: SignupData
)

data class SignupData(
    val memberId: Long,
    val email: String,
    val name: String,
    val createdAt: String?
)